extensions/wikia/VideoHandlers/apiwrappers/YoutubeApiWrapper.class.php

```diff
@@ -90,9 +86,8 @@ public function getThumbnailUrl() {
 	 * @return array
 	 */
 	protected function getVideoThumbnails() {
-		if ( !empty($this->interfaceObj['entry']['media$group']['media$thumbnail']) ) {
-
-			return $this->interfaceObj['entry']['media$group']['media$thumbnail'];
+		if ( !empty( $this->interfaceObj['items'][0]['snippet']['thumbnails'] ) ) {
```

Comment:
from what I see we are operating on:
```php
$this->interfaceObj['items'][0]
```
how about setting
```php
$this->interfaceObj = $this->interfaceObj['items'][0]
```
it would clean up all those nestings a bit